### PR TITLE
feat(file-reporter): remove file reporter

### PR DESCRIPTION
**Pull request by william2958**
Resolves issue https://github.com/pnpm/pnpm/issues/4949 where pnpm install will crash in a very large repository as the logs object in memory becomes too large for JSON.stringify to handle.

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -14653,4 +14604,8 @@ packages:
       - encoding
       - supports-color
     dev: true
-    patched: true
+
+patchedDependencies:
+  pkg@5.7.0:
+    hash: pp6fkuhwkrqq7cjcj7uqpf37e4
+    path: patches/pkg@5.7.0.patch
```
Comment (zkochan) [inline pnpm-lock.yaml]: do you use the latest pnpm? The lockfile doesn't look right

File: .changeset/rare-walls-smell.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"pnpm": minor
```
Comment (zkochan) [inline .changeset/rare-walls-smell.md]:
```suggestion
"pnpm": patch
```

let's make it a patch

File: .changeset/rare-walls-smell.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Remove file reporter logging. Logged file is not useful.
```
Comment (zkochan) [inline .changeset/rare-walls-smell.md]:
```suggestion
Remove file reporter logging. Logged file is not useful [#4949](https://github.com/pnpm/pnpm/issues/4949).
```

File: pnpm-lock.yaml (hunk 1 above)
Comment (william2958) [reply pnpm-lock.yaml]: It seems I was using a previous version of pnpm to install (7.1.7) but after upgrading to the latest version (7.5.0) & rerunning pnpm install, the pnpm-lock.yaml file is unchanged - is there something else I missed?

**Pull request merged**